index.js

```diff
@@ -78,6 +78,12 @@ class ServerlessPlugin {
                         shortcut: 'a',
                         type: 'string',
                     },
+                    function: {
+                        usage: 'The name of the function to invoke',
```

Comment:
```suggestion
                        usage: 'The name of the function to invoke (optional, auto-discovered by default)',
```

just to avoid any confusion for users